retorchfiles/scripts/testexecution.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 #https://stackoverflow.com/questions/4365418/get-state-of-maven-execution-within-shell-script
 $E2ESUITE_URL/retorchfiles/scripts/writetime.sh "$2" "$1" #$2 is the stage, $1 tjob
-echo 'Starting 10 seconds sleep'
-sleep 10
-echo 'End 10 seconds sleep'
-mvn test
+mvn test -DSUT_URL="156.35.119.57" -DSUT_PORT=5100 -Ddirtarget=TJOBExample -Dtjob_name=tjobe
```

Comment:
@javiertuya  What is the most convenient way to store the server URL? one Jenkins credential? Create an env variable in the slave dockerfile?

Reply:
- Jenkins credential: this would require some jenkins configuration, but this should be independent from the agent, I would discard this
- Environment variable in the agent: right, it is similar to other configurations you made, where you use an env var that fallbacks to localhost when running running in local
- Another option: use docker addressing, if the SUT network is added to the jenkins agent

Reply:
The issue has been addressed in the latest commit. I am employing Docker local networks, which are polled to the various containers by the slave, along with binded ports. This enables us to deploy multiple System Under Tests (SUTs) in the future. Instead of relying on container names, which caused issues due to their diverse origins as demonstrated in #13, we now utilize a pool of ports that will be distributed among the various scenarios.